diff --git a/node_modules/next/dist/export/index.js b/node_modules/next/dist/export/index.js
index b9bac4c..24d6fb6 100644
--- a/node_modules/next/dist/export/index.js
+++ b/node_modules/next/dist/export/index.js
@@ -315,8 +315,8 @@ async function exportApp(dir, options, span) {
             };
         }
         const { i18n , images: { loader ="default" , unoptimized  }  } = nextConfig;
-        if (i18n && !options.buildExport) {
-            throw new ExportError(`i18n support is not compatible with next export. See here for more info on deploying: https://nextjs.org/docs/messages/export-no-custom-routes`);
+        if (i18n && !options.buildExport && !i18n.defaultLocale) {
+            throw new Error(`i18n support is not compatible with next export. See here for more info on deploying: https://nextjs.org/docs/deployment`);
         }
         if (!options.buildExport) {
             const { isNextImageImported  } = await nextExportSpan.traceChild("is-next-image-imported").traceAsyncFn(()=>_fs.promises.readFile((0, _path.join)(distDir, _constants1.EXPORT_MARKER), "utf8").then((text)=>JSON.parse(text)).catch(()=>({})));
